app.rb

```diff
@@ -0,0 +1,323 @@
+require_relative './class/music_album'
+require_relative './class/genre'
+require_relative './class/book'
+require_relative './class/label'
+require_relative './class/game'
+require_relative './class/author'
```

Comment:
- [ ] At the method, importing these modules returns an error that is caused by wrong naming. The folder name `Class` and not `class`. Please kindly help fix this.

Reply:
Thanks! It's done!